test/orange/g4org/SolidConverter.test.cc

```diff
@@ -156,7 +156,7 @@ TEST_F(SolidConverterTest, cons)
 {
     this->build_and_test(
         G4Cons("Solid TubeLike #1", 0, 50, 0, 50, 50, 0, 360),
-        R"json({"_type":"shape","interior":{"_type":"cylinder","halfheight":5.0,"radius":5.0},"label":"Solid TubeLike #1"})json",
+        R"json({"_type":"shape","interior":{"_type":"cone","halfheight":5.0,"radii":[5.0,5.0]},"label":"Solid TubeLike #1"})json",
```

Comment:
This is to simplify the polycone construction; instead of having to construct a special case, we might as well "identify" it as a degenerate cylinder if that's what the user's asked for....